src/main/java/edu/ie3/datamodel/models/input/container/InputContainer.java

```diff
@@ -30,5 +30,7 @@ interface InputContainerCopyBuilder<T extends UniqueInputEntity> {
 
     /** Returns the altered {@link InputContainer} */
     InputContainer<T> build() throws ValidationException;
+
+    InputContainerCopyBuilder<T> thisInstance();
```

Comment:
We could change this interface to be an abstract class. If we do, we could make the method `thisInstance()` protected.